packages/create-amplify/src/tsconfig_initializer.ts

```diff
@@ -53,10 +55,23 @@ export class TsConfigInitializer {
     }
 
     try {
-      await this.execa('npx', tscArgs, {
-        stdio: 'inherit',
+      let aggregatedStdout = '';
+      const aggregatorStream = new stream.Writable();
+      aggregatorStream._write = function (chunk, encoding, done) {
+        aggregatedStdout += chunk;
+        done();
+      };
+
+      const childProcess = this.execa('npx', tscArgs, {
+        stdin: 'inherit',
         cwd: this.projectRoot,
       });
+
+      childProcess?.stdout?.pipe(aggregatorStream);
+      childProcess?.stderr?.pipe(aggregatorStream);
+
+      await childProcess;
+      await this.logger.debug(aggregatedStdout);
```

Comment:
if child process fails, we will miss printing the `stderr` to the console.

Reply:
Done in [52a5eda](https://github.com/aws-amplify/amplify-backend/pull/595/commits/52a5eda31ca1d6dc5bfdca59e500c1ed746c0669)